code/scripting/scripting.cpp

```diff
@@ -104,6 +104,7 @@ static USED_VARIABLE SCP_vector<std::shared_ptr<BuiltinHook>> Script_actions
 	std::make_shared<BuiltinHook>("On Subsystem Destroyed",	CHA_ONSUBSYSDEATH ),
 	std::make_shared<BuiltinHook>("On Goals Cleared",		CHA_ONGOALSCLEARED ),
 	std::make_shared<BuiltinHook>("On Briefing Stage",		CHA_ONBRIEFSTAGE ),
+	std::make_shared<BuiltinHook>("On Debris Created",		CHA_ONDEBRISCREATED),
```

Comment:
There is a new system for defining hooks that allows to specify documentation about it. See for example `OnMessageReceivedHook` for an example.

I really should add a comment here explaining that...

Reply:
Ok, changed.